recipes/datasets/plugins/plugin_consensus.py

```diff
@@ -0,0 +1,66 @@
+from kili.plugins import PluginCore
+from kili.types import Label
+
+
+class PluginHandler(PluginCore):
+    """
+    Custom plugin instance
+    """
+
+    def aggregate_object_detection_label(self, object_job_key, labels_array):
+
+        json_response_array = {}
+
+        for key in object_job_key:
+            l_key = []
+            for _, label in enumerate(labels_array):
+                for annotation in label["jsonResponse"][key]["annotations"]:
```

Comment:
I think this will fail if there is a label that does not have an annotation for each job (for example if a job is not required), check that `key` is in `label["jsonResponse"]`